src/en/about-juju.md

```diff
@@ -2,12 +2,11 @@ Title: What is Juju?
 
 # What is Juju?
 
-Juju is a state-of-the-art, open source, universal modelling tool for service
-oriented architectures and application oriented deployments. Juju allows you to
-deploy, configure, manage, maintain, and scale cloud applications quickly and
-efficiently on public clouds, as well as on physical servers, OpenStack, and
-containers. You can use Juju from the command line or through its beautiful
-GUI.
+Juju is a state-of-the-art, open source modelling tool for operating software
+in the cloud. Juju allows you to deploy, configure, manage, maintain, and
+scale cloud applications quickly and efficiently on public clouds, as well as
+on physical servers, OpenStack, and containers. You can use Juju from the
+command line or through its beautiful GUI.
 
```

Comment:
Ok, I understand but I also think that 

    service-oriented architectures and application oriented deployments

Doesn't speak that clearly and directly to the value of Juju and I'd like to make this change. If I need signoff from someone I'll go get it.